Lesson27/Merve/update.js

```diff
@@ -0,0 +1,51 @@
+const urlParams = new URLSearchParams(window.location.search);
+const postId = urlParams.get('postId');
+
+// 1. Fetch post data for the post with given postId
+function getPostById(id) {
+  return fetch(`${URL}${id}`)
```

Comment:
URL variable is not present in this file. Either create a `const URL = 'https://jsonplaceholder.typicode.com/posts/'``
or update the current line like this:
```suggestion
  return fetch(`https://jsonplaceholder.typicode.com/posts/${id}`)
```